src/pudl/transform/vcerare.py

```diff
@@ -226,6 +230,55 @@ def _check_for_valid_counties(
     return df
 
 
+def _handle_2015_nulls(combined_df: pd.DataFrame, year: int):
+    """Handle unexpected null values in 2015.
+
+    In 2015, there are a few hundred null values for PV capacity factors that
+    should be zeroed out, according to correspondence with the data provider.
+    This function narrowly zeroes out these nulls, expecting that the rest of the
+    data should conform to the expectation of no-null values.
+    """
+    if year == 2015:
+        logger.info(
+            f"{len(combined_df.loc[(combined_df.capacity_factor_solar_pv.isnull()) & (combined_df.report_year == '2015')])} null PV capacity values found in the 2015 data. Zeroing out these values."
+        )
```

Comment:
Optional:

would prefer 3x+ repeated selectors (here and in `_clip_unexpected_2016_pv_capacity`) get pulled out for reuse rather than copy-pasted, that way it's easier to see at a glance that they are all doing the same thing. 

it would need to be re-spaced to appease the pre-commit gods, but this change would start like:

```suggestion
        null_selector = (combined_df.capacity_factor_solar_pv.isnull()) & (combined_df.report_year == '2015')
        logger.info(
            f"{len(combined_df.loc[null_selector])} null PV capacity values found in the 2015 data. Zeroing out these values."
        )
```